scripts/print_enclave_hash

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o xtrace
+
+# Prints the SHA1 hash of the trusted enclave code.
```

Comment:
Are we going to need this script on its own?  If not, could just make if a function in `check_enclave_reproducibility`.

Reply:
I think it is useful to compare hashes across machines / users, as we have just done on Slack :)